src/frontend/src/pages/PIAIntakeForm/index.tsx

```diff
@@ -472,7 +473,59 @@ const PIAFormPage = () => {
 
     return () => clearTimeout(autoSaveTimer);
   });
+  const handleBack = () => {
+    navigate(-1);
+  };
+  const handleNext = () => {
+    const tabName = pathname.split('/')[3];
 
+    switch (tabName) {
+      case 'intake':
+        navigate(pathname.replace('intake', 'collectionUseAndDisclosure'));
+        break;
+      case 'collectionUseAndDisclosure':
+        navigate(
+          pathname.replace(
+            'collectionUseAndDisclosure',
+            'storingPersonalInformation',
+          ),
+        );
+        break;
+      case 'accuracyCorrectionAndRetention':
+        navigate(
+          pathname.replace(
+            'accuracyCorrectionAndRetention',
+            'agreementsAndInformationBanks',
+          ),
+        );
+        break;
+      case 'securityPersonalInformation':
+        navigate(
+          pathname.replace(
+            'securityPersonalInformation',
+            'accuracyCorrectionAndRetention',
+          ),
+        );
+        break;
+      case 'agreementsAndInformationBanks':
+        navigate(
+          pathname.replace('agreementsAndInformationBanks', 'additionalRisks'),
+        );
+        break;
+      case 'storingPersonalInformation':
+        navigate(
+          pathname.replace(
+            'storingPersonalInformation',
+            'securityPersonalInformation',
+          ),
+        );
```

Comment:
Instead of bluntly changing via strings, can you check if we can leverage `piaFormSideNavPages` here?

Reply:
Ideally, this should be calculated based on piaFormsSidenavpages. 
Find the current page, search for it in piaFormsidenavpages and get the index. if index is 0 there is no back if index is length of the string no next, etc
anything in between have prev to index-1 and next to index+1 

I would think through a state-machine and design it form that POV by expanding on piaFormsSideNAvPage object . my 2 cents.

Reply:
let me know your thought about the new change,thanks